runtime/service/npud/core/Server.cc

```diff
@@ -33,7 +33,7 @@ Server::Server() noexcept
 
 void Server::run(void)
 {
-  std::cout << "Starting Server" << std::endl;
+  VERBOSE(Server) << "Starting Server\n";
```

Comment:
I have a question. Why did you change `std::endl` to `\n`? Was there a problem?

Reply:
No, there is no problem. I just follow different code rules.
One runtime uses both of `"blah blah ~~\n"` and `"blah bla~~" << std::endl`.

Reply:
I was wondering if there was a problem. I'm happy because we would have to change the other codes like this in `onert` if there was a problem. But I think that `std::endl` is better than `\n` because `std::endl` flushes the stream. Anyway this change doesn't matter.